environment.yml

```diff
@@ -0,0 +1,6 @@
+name: anaconda-client-env
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)